File: src/examples/python/trackers/retrieve_trackers.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_TEST_API_KEY')

trackers = easypost.Tracker.all(
    # Optionally filter trackers by page size and date range
    page_size=100,
    start_datetime='2020-03-01T08:50:00Z',
    end_datetime='2020-03-02T08:50:00Z',
)

print(trackers)

File: src/examples/node/trackers/retrieve-trackers.js
const Easypost = require('@easypost/api');
const dotenv = require('dotenv');

dotenv.config({ path: '/Users/jhammond/git/easypost/easypost-tools/.env' });
const api = new Easypost(process.env.EASYPOST_TEST_API_KEY);

// Grab all trackers in date range
api.Tracker.all({
  start_datetime: "2020-03-01T08:50:00Z",
  end_datetime: "2020-03-27T08:50:00Z",
}).then(console.log).catch(console.log);
